vivarium/core/serialize.py

```diff
@@ -155,24 +70,7 @@ def deserialize(self, data: Any) -> List[Any]:
 class DictSerializer(Serializer):  # pylint: disable=abstract-method
 
     def __init__(self) -> None:
-        super().__init__(exclusive_types=(dict,))
-
-    def can_serialize(self, data: Any) -> bool:
-        return isinstance(data, dict)
-
-    def serialize(self, data: dict) -> dict:
-        '''Serialize to dict of serialized elements.
-
-        Note that dict serialization of keys is lossy. For example, a
-        float key will be serialized as a string, which will then be
-        deserialized to a string.
-        '''
-        serialized = {}
-        for key, value in data.items():
-            if not isinstance(key, str):
-                key = str(key)
-            serialized[key] = serialize_value(value)
-        return serialized
+        super().__init__()
```

Comment:
You can just omit this `__init__()` right?

Reply:
Went through and removed all of these in 5394975.